src/bci_build/package/basecontainers.py

```diff
@@ -35,6 +35,145 @@ def _get_eula_package_names(os_version: OsVersion) -> tuple[str, ...]:
     return ("skelcd-EULA-bci",)
 
 
+def _get_base_config_sh_script(os_version: OsVersion) -> str:
+    return textwrap.dedent(
+        r"""
```

Comment:
The dedent does nothing
```suggestion
    return r"""
```

Reply:
replaced with jinja.template rendering.